### ISSUE TITLE
Fails when declaring WooCommerce as a dependency

### ISSUE THREAD

**Issue by rafark** [WordPress/plugin-check-action#611]
`* Requires Plugins:  woocommerce` is in my plugin header. When this action runs it fails activating that plugin because woo seems to generate warnings. I added a .wp-env.override.json to my repo root to suppress the warnings but this actions seems to ignore that. If i manually install wp-env in the action instead of using the plugin-check-action then woocommerce installs correctly.

// .wp-env.override.json, only works when installing and calling wp-env manually instead of this action

```
{
          "core": "https://wordpress.org/wordpress-${{ steps.wp-version.outputs.wpVersion }}.zip",
          "config": {
            "WP_DEBUG": true,
            "WP_DEBUG_DISPLAY": false,
            "WP_DEBUG_LOG": true
          }
}

```

  ℹ Starting 'wp plugin activate build' on the cli container. 
  Notice: Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>woocommerce</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href="https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /var/www/html/wp-includes/functions.php on line 6170
  Warning: Failed to activate plugin. The plugin generated unexpected output.
  Error: No plugins activated.
  ✖ Command failed with exit code 1
  Command failed with exit code 1

**Comment by swissspidy** [comment]
I recommend reporting this to them as well, their code should not produce warnings

**Comment by rafark** [comment]
No, it was my fault, my dummy (dummy as in test, for testing) plugin had an echo statement and that was causing the activation issue. I just removed that. Sorry.

WooCommerce must have some hooks that trigger when plugin_sandbox_scrape() fails and that causes woocommerce to call internationalization functions before 'init', so it is a woocommerce bug most likely since its translation functions should not be called on plugin_sandbox_scrape because its too early hence the Notice: Function _load_textdomain_just_in_time was called incorrectly.

I might open an issue there.
